projects/core/package.json

```diff
@@ -22,7 +22,7 @@
     "@ngrx/effects": "^14.3.0",
     "@ngrx/router-store": "^14.3.0",
     "@ngrx/store": "^14.3.0",
-    "angular-oauth2-oidc": "^13.0.1",
+    "angular-oauth2-oidc": "^14.0.1",
```

Comment:
I think that updating a major version of a dependency in a minor release might be considered non backwards compatible (a breaking change).  Let's discuss.

Reply:
When we upgrade angular to v14, we should update this lib as well. But, `angular-oauth2-oidc` is not ready yet at that time.

Reply:
See https://wiki.one.int.sap/wiki/display/PSSEC/SDOL-005